src/fluree/db/ledger/indexing.clj

```diff
@@ -197,7 +198,7 @@
          (xf))
 
         ;; Iteration:
-        ;;   1. Incorporate each successive node with it's corresponding novelty
+        ;;   1. Incorporate each successive node with its corresponding novelty
```

Comment:
Ope, you fixed it. That's what I get for going through in commit order :1st_place_medal: